File: Pascal/Lab1/Variant6.pas
Program lab1;
Uses Crt;
Var a,b,t,Formula: real;
Begin
  Write('Введите первое число: '); Readln(a);
  Write('Введите второе число: '); Readln(b);
  Write('Введите третье число: '); Readln(t);
  ClrScr;
  Formula:=b*sin(a*sqr(t)*cos(2*t))-1;
  Writeln('Вариант 6.  ',Formula);
End.

File: Python/Lab1/Variant6.py
from math import *
a = float(input("Введите значение a: "))
b = float(input("Введите значение b: "))
t = float(input("Введите значение t: "))
s = b*sin(a*t*t*cos(2*t))-1
print("\nЗначение s =", s)
